release_management/_utils.py

```diff
@@ -149,12 +149,13 @@ def create_release(repository, tag, release_version, changelog_url):
     return release
 
 
-def vendor(manifest_abs_path, release_version):
+def vendor(manifest_abs_path, release_version, suffix=None):
```

Comment:
Add "*" here as: ```release_version, *, suffix=None```. By Python rules, that makes it a mandatory keyword argument. Without that ```*```, it can be either a keyword argument or a positional argument, which has historically been a source of confusion for many Python programs that couldn't make the distinction.

Reply:
If I do that, these lint errors appear:
```
pylint ./release_management/*.py --disable=R0801
************* Module create_artifacts
release_management/create_artifacts.py:102:30: E1121: Too many positional arguments for function call (too-many-function-args)
release_management/create_artifacts.py:115:30: E1121: Too many positional arguments for function call (too-many-function-args)
```

Reply:
That's because the requested change makes the parameter a mandatory keyword argument. So you can't pass it as a positional argument, you have to pass it as a keyword argument.

Reply:
OK, I think I finally figured it out.  Updated via force-push.